src/intellisense/pythonKernelCompletionProvider.ts

```diff
@@ -218,21 +220,8 @@ export class PythonKernelCompletionProvider implements CompletionItemProvider {
         cancelToken: CancellationToken
     ) {
         const notebook = getAssociatedJupyterNotebook(document);
-        if (notebook) {
```

Comment:
Moved this to a node specific implementation. In the web we don't start pylance so we can't ask it for it's completions. 

Hopefully we'll be able to fix this later.